Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -83,9 +85,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $ui = new SymfonyStyle($input, $output);
 
-        /** @var ManagerRegistry $doctrine */
-        $doctrine = $this->getContainer()->get('doctrine');
-        $em       = $doctrine->getManager($input->getOption('em'));
+        // @todo The method_exists call can be removed once the DoctrineBundle dependency has been bumped to at least 1.10
+        if (method_exists($this, 'getDotrine')) {
```

Comment:
`getDotrine`? 😏

Reply:
Oh come on...

Reply:
Ouch... also didn't see this :S

Reply:
Fixed 🙄

Reply:
@Ocramius I'm still working on tests that manage to get to that point in the functionality. Will most likely end with completely changing the command in 4.0...